scripts/lint-extras.sh

```diff
@@ -59,13 +59,19 @@ function lint_helm() {
     run_check '' helm lint --namespace=test charts/*
 }
 
+function lint_py() {
+    # Install via: pip install pylint
+    run_check '.*\.py' pylint -s n -f colorized
```

Comment:
Ideally run commands with long options, like `--score=y` and so on. People reading the short options will not know what they stand for, long options are a little more user-friendly.

Reply:
I'll keep that in mind. Thanks!